src/quacc/runners/ase.py

```diff
@@ -362,3 +369,171 @@ def _set_sella_kwargs(self, optimizer_kwargs: dict[str, Any]) -> None:
 
         if not self.atoms.pbc.any() and "internal" not in optimizer_kwargs:
             optimizer_kwargs["internal"] = True
+
+
+def run_neb(
+    images,
+    relax_cell: bool = False,
+    fmax: float = 0.01,
+    max_steps: int | None = 1000,
+    optimizer: NEBOptimizer | BFGS = NEBOptimizer,
+    optimizer_kwargs: OptimizerKwargs | None = None,
+    run_kwargs: dict[str, Any] | None = None,
+    neb_kwargs: dict[str, Any] | None = None,
+    copy_files: SourceDirectory | dict[SourceDirectory, Filenames] | None = None,
+) -> list[Atoms]:
+    """
+    Run NEB optimization.
+
+    Parameters
+    ----------
+    images : list of Atoms
+        List of images representing the initial path.
+    relax_cell : bool, optional
+        Whether to relax the unit cell shape and volume.
+    fmax : float, optional
+        Tolerance for the force convergence (in eV/A).
+    max_steps : int or None, optional
+        Maximum number of steps to take.
+    optimizer : Optimizer or BFGS, optional
+        Optimizer class to use.
+    optimizer_kwargs : dict or None, optional
+        Dictionary of kwargs for the optimizer.
+    run_kwargs : dict or None, optional
+        Dictionary of kwargs for the run() method of the optimizer.
+    neb_kwargs : dict or None, optional
+        Dictionary of kwargs for the NEB.
+    copy_files : str or dict or None, optional
```

Comment:
Please update the docstring format to match the rest of quacc.

Reply:
Did that.